### ISSUE TITLE
Upkeep order keeps crashing

### ISSUE THREAD

**Issue by YoungShinji** [P3pp3rF1y/AncientWarfare2#307]
Hello,

Everytime when i try to right click the Upkeep order it keeps crashing.
I'm on a resonant rise mainline server

is there a solution ?

**Comment by bencrew** [comment]
i am also having the same issue. the order crashes only after it has been bound with something

**Comment by GotoLink** [comment]
Probably duplicate of #297
Missing crash report to make a difference.

**Comment by YoungShinji** [comment]
Crash report > http://pastebin.com/AcMtU6pd

**Comment by GotoLink** [comment]
Fixed in build 105.
